minifi/minifi-c2/minifi-c2-jetty/src/main/java/org/apache/nifi/minifi/c2/jetty/JettyServer.java

```diff
@@ -134,7 +182,11 @@ private static WebAppContext loadWar(final File warFile, final String contextPat
         // configure the max form size (3x the default)
         webappContext.setMaxFormContentSize(600000);
 
-        webappContext.setClassLoader(new WebAppClassLoader(parentClassLoader, webappContext));
+        try {
+            webappContext.setClassLoader(new WebAppClassLoader(parentClassLoader, webappContext));
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
```

Comment:
Is this try-catch necessary?

Reply:
Yes, otherwise I couldn't call the method within lambda as it throws a checked IO exception. I'm adding an error message to the exception.